.github/review-bot.yml

```diff
@@ -59,39 +63,20 @@ rules:
     teams:
       - core-devs
 
-  # cumulus
-  - name: Runtime files cumulus
-    countAuthor: true
-    condition:
-      include:
-        - ^cumulus/parachains/runtimes/assets/(asset-hub-kusama|asset-hub-polkadot)/src/[^/]+\.rs$
-        - ^cumulus/parachains/runtimes/bridge-hubs/(bridge-hub-kusama|bridge-hub-polkadot)/src/[^/]+\.rs$
-        - ^cumulus/parachains/runtimes/collectives/collectives-polkadot/src/[^/]+\.rs$
-        - ^cumulus/parachains/common/src/[^/]+\.rs$
```

Comment:
Should `common` not still be in here since the `polkadot` and `kusama` system parachain runtimes still depend on this crate?
I recently started the process of separating testnet configs into a different package for this reason, so that **only** `polkadot` and `kusama` changes are caught by this

Reply:
Makes sense, I'll remove that testnet common package and bring them back into common